assets/css/utilities/_colors.scss

```diff
@@ -58,6 +58,9 @@ $security-green: $jade;
 $security-yellow: $dandelion;
 $bright-green:  $jade !important; //override Security Center nested css selectors
 
+//opacity
+.opacity-5 { opacity: .5; }
```

Comment:
Added to blockchain-css https://github.com/blockchain/brand-assets/blob/css/sass/utilities/_colors.scss#L16-L20. Please remove this.